source/_posts/2019-03-08-shipit-xiv.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "ShipIt XIV"
+author: Caitie Oder
+date: 2019-03-08
+tags: [culture, engineering, hackathon, shipit]
+---
+
+Readily awaiting the winter storm racing towards Kansas City, ShipIt Day XIV teams began preparing their ideas and tactics to compete in the 24-hour hackathon competition. Although the snowy weather may have kept participants between the warm walls of our Innovations Campus, the winter storm did not disturb the initiative and enthusiasm of the fourteen participating teams.
```

Comment:
> Readily awaiting the winter storm racing towards Kansas City, ShipIt Day XIV teams began preparing their ideas and tactics to compete in the 24-hour hackathon competition.

I would avoid this entry statement, as it also seems like something bad is coming, then the statement flexes to our event. Perhaps:

> It is winter time in Kansas City, which also included our fourteenth edition of ShipIt, our routine hackathon competition we host at Cerner Engineering. Although the snowy weather may have kept participants between the warm walls of our Innovations Campus, the winter storm did not disturb the initiative and enthusiasm of the fourteen participating teams.

Reply:
Updated to suggested introduction